helper-cli/src/main/kotlin/commands/repoconfig/RemoveEntriesCommand.kt

```diff
@@ -95,7 +95,7 @@ internal class RemoveEntriesCommand : CliktCommand(
 
         val scopeExcludes = ortResult
             .getProjects()
-            .flatMap(ortResult.dependencyNavigator::scopeNames)
+            .flatMap { project -> project.scopes.map { scope -> scope.name } }
```

Comment:
While this works, wouldn't it make sense in the long run to drop the conversion to the tree format (in `readOrtResult()`), especially as the graph format is the default now?

I think, the problem here is that no `DependencyNavigator` can be created, although the `OrtResult` is valid. We already had this discussion in the past.

Reply:
> While this works, wouldn't it make sense in the long run to drop the conversion to the tree format (in readOrtResult()), especially as the graph format is the default now?

Probably yes. For now I need a short term solution, just fixing up that crash. So I believe that's out of scope.